### PR TITLE
MON-3530: Update OWNERS

### PR DESCRIPTION
**Pull request by simonpasquier**

### CODE REVIEW

**Review by marioferh — COMMENTED** [review]
/lgtm

File: OWNERS [hunk 1]
```diff
@@ -4,23 +4,29 @@ reviewers:
 - BlueShells
```
Comment (rexagod) [inline OWNERS:4]:
```suggestion
```
Shows up with the "deactivated" suffix on Slack, may be removed.

**Comment by raptorsun** [comment]
created a ticket for this https://issues.redhat.com/browse/MON-3530
/retitle MON-3530: Update OWNERS

**Pull request merged**